Sources/race.swift

```diff
@@ -91,7 +91,6 @@ public func race<U: Thenable>(fulfilled thenables: [U]) -> Promise<U.T> {
                     }
                 case .fulfilled(let value):
                     guard rp.isPending else { return }
-                    countdown = 0
```

Comment:
An unnecessary line.

Reply:
Probably, but there is no reason to remove it. It is possible this is required. Please leave it.